internal/ingress/controller/controller_test.go

```diff
@@ -188,7 +189,7 @@ func TestCheckIngress(t *testing.T) {
 			},
 		},
 	}
-
+	/* TODO: Right now we have stopped to check IngressClass in Admission
```

Comment:
is this because of the older k8s version diffs and issues with kind etc.?

Reply:
Sort of...validating IngressClass in Admission is hard, as we don't know what ingress class that annotation represents.

https://github.com/kubernetes/ingress-nginx/pull/7341/files/94f3838205e048f62f5d3fa8b60bb2caa0af6b45#diff-4198ec010671801881244a8052177f31bcbc682c99fbd7391bceb136025c0568R225-R229